web/src/RootUser.jsx

```diff
@@ -92,7 +114,7 @@ export default function RootUser() {
   return (
     <>
       <Button variant="link" onClick={open}>
-        {rootLabel()}
+        {rootLabel().concat(SSHKeyLabel())}
```

Comment:
It is OK. If you prefer, you can do `${rootLabel()} ${SSHKeyLabel()}` (so you can avoid having to add that tricky whitespace to the end of `rootLabel()`.

Reply:
will try, thanks

Reply:
Oh, sorry, the formatting makes my suggestion not clear:

```javascript
`${rootLabel()} ${SSHKeyLabel()}`
```

I was referring to [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).